Sources/JSExtension.swift

```diff
@@ -0,0 +1,338 @@
+//
```

Comment:
This is still one big file, please split this up as mentioned before, one file for each logical block. For example, move the JS wrappers to another file.